app/views/posts/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Posts#index</h1>
-<p>Find me in app/views/posts/index.html.erb</p>
+<h1>Posts</h1>
+<hr>
+  <div>
+  <% @posts.each do |post| %>
+    <p>
+      <h2><%= post.title %></h2>
+      <div>
+        Created at: <%= post.created_at.strftime('%d/%m/%Y %H:%M') %>  |
+        Submitted by: <%= post.user.forename %> <%= post.user.surname %>
+      </div>
+      <br>
+      <%= strip_tags(post.body.to_s.truncate(300)) %>
```

Comment:
Consider refactoring logic out of the view, which will keep it cleaner and easier to read and test in the long run